parlai/tasks/woz/build.py

```diff
@@ -22,13 +44,8 @@ def build(opt):
         build_data.make_dir(dpath)
 
         # Download the data.
-        fnames = ['woz_test_en.json', 'woz_train_en.json', 'woz_validate_en.json']
-        for fname in fnames:
-            url = (
-                'https://github.com/nmrksic/'
-                'neural-belief-tracker/raw/master/data/woz/' + fname
-            )
-            build_data.download(url, dpath, fname)
+        for donwloadable_file in RESOURCES:
```

Comment:
spelling?

Reply:
Yup, that was dumb, I'll go through and change them